PROTO_tests/tests/message_test.py

```diff
@@ -48,17 +48,20 @@ def test_request_msg(self):
         assert msg != msgNonDupe
 
         assert (
-            "RequestMessage(user='user1', token='token', cmd='connect', format=STRING, args=None, size=-1)"
+            "RequestMessage(user='user1', token='token', "
+            "cmd='connect', format=STRING, args=None, size=-1)"
             == str(msg)
         )
 
         assert (
-            "RequestMessage(user='user1', token='token', cmd='connect', format=STRING, args=None, size=-1)"
+            "RequestMessage(user='user1', token='token', "
```

Comment:
this seemed basically the same in all these cases so i rewrote the function
```python
def test_request_msg(self):
    msg = RequestMessage(user="user1", token="token", cmd="connect", format=MessageFormat.STRING)
    msgDupe = RequestMessage(user="user1", token="token", cmd="connect", format=MessageFormat.STRING)
    msgNonDupe = RequestMessage(
        user="user1", token="token", cmd="connect", format=MessageFormat.BINARY
    )
    min_msg = RequestMessage(user="user1", cmd="connect")

    assert "user1" == msg.user
    assert "token" == msg.token
    assert "connect" == msg.cmd
    assert MessageFormat.STRING == msg.format

    assert msg == msgDupe
    assert msg != msgNonDupe

    rep_msg = (
        "RequestMessage(user='user1', token={}, cmd='connect', format=STRING, args=None, size=-1)"
    )
    assert rep_msg.format("'token'") == str(msg)
    assert rep_msg.format("'token'") == repr(msg)
    assert rep_msg.format("None") == str(min_msg)
    assert rep_msg.format("None") == repr(min_msg)

    dict_msg = (
        '{{"user": "user1", "token": {}, "cmd": "connect", "format": "STRING", "args": "",'
        ' "size": -1}}'
    )
    assert dict_msg.format('"token"') == json.dumps(msg.asdict())
    assert dict_msg.format('""') == json.dumps(min_msg.asdict())
    assert json.loads(json.dumps(msg.asdict())) == msg.asdict()
```

Reply:
Updated